app/controllers/api/v1/base.rb

```diff
@@ -1,19 +1,13 @@
 # Base controller for all API controllers
 class Api::V1::Base < ApplicationController
-
-  before_filter :authenticate_user!
   respond_to :json
-  helper_method :inherited_collection
 
   protected
-
-  # collection is overwritten by RABL
-  def inherited_collection
-    collection
-  end
-
   def check_write_permission
     authorize! :write, resource
   end
 
+  def bullshit_test
+    'foofum'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.